src/platyPS/platyPS.psm1

```diff
@@ -1777,3 +1777,19 @@ function Convert-PsObjectsToMamlModel
     return $MamlCommandObject
 }
 #endregion
+
+#region Parameter Auto Completers
+# Completers are storted in .\psplaty.AutoCompleters.ps1
+
+# Add all completers in this IF block to ensure TabExpansionPlusPlus is installed, else there is no -Description Param in the V5 Register-ArgumentCompleter Command.
+if (Get-Command -Name Register-ArgumentCompleter -Module TabExpansionPlusPlus -ErrorAction Ignore) {
```

Comment:
The only thing i have against the PS v5 method, is that it works poorly.  I think if they have TabExpansionPlusPlus installed, use it, but if not, then do not add any tab completers.